### ISSUE TITLE
 Update nb selected row whene rows(selector).remove()

### ISSUE THREAD

**Issue by Thibault34** [gyrocode/jquery-datatables-checkboxes#26]
Hello,
When I use table.rows('.myClass').remove(), the number of select row is not updated.

I must be used :
table.rows('.myClass').deselect().remove().draw();

The problem isn't in default datatable checkbox selection :
http://live.datatables.net/seqejati/4/edit

Can you help me please ?

**Comment by mpryvkin** [comment]
Good catch! However the example you mentioned uses different way to show checkboxes. 

For now you need `deselect()` as shown below. See [this example](https://jsfiddle.net/47jzysg9/) for code and demonstration.

    table.rows('.myClass').deselect().remove().draw();

I will see if I could find a way to catch row removal event and update internal data accordingly.

**Comment by Thibault34** [comment]
Thank you
